content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -50,27 +46,35 @@ NOTE: If you discover a previously unreported issue, please let us know: read li
 
 == Javax XML Bind libraries
```

Comment:
The proper name for JAXB is either "Java Architecture for XML Binding" (old) or "Jakarta XML Binding" (new) ([Wikipedia](https://en.wikipedia.org/wiki/Jakarta_XML_Binding))

Reply:
good to know, thanks @basil I've updated the header to reflect the new proper name.